easybuild/easyconfigs/z/zsh/zsh-5.8-foss-2019b.eb

```
@@ -0,0 +1,18 @@
+easyblock = 'ConfigureMake'
+
+name = 'zsh'
+version = '5.8'
+
+homepage = 'http://www.zsh.org/'
+description = "Zsh is a shell designed for interactive use, although it is also a powerful scripting language."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

I would suggest to change the toolchain to `GCC` instead of `foss` as zsh does not use BLAS/LAPACK or MPI.
```suggestion
toolchain = {'name': 'GCC', 'version': '8.3.0'}
```